models/topics.model.js

```diff
@@ -56,11 +56,21 @@ exports.getCommentsDataByArticleId = (id) => {
 exports.addCommentToArticle = (id, obj) => {
   if (Object.keys(obj).length !== 2 || !obj.username || !obj.body) {
     return Promise.reject({ status: 400, msg: "Bad request" });
-  } 
+  }
   return this.getArticleDataById(id).then((article) => {
     return db.query(
       "INSERT INTO comments (body, author, article_id, votes) VALUES ($1, $2, $3, $4) RETURNING *",
       [obj.body, obj.username, id, 0]
     );
   });
 };
+
+exports.patchArticle = (id, body) => {
+  if(Object.keys(body).length !==1 || typeof body.inc_votes !== 'number'){
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  return this.getArticleDataById(id).then((result) => {
+    result.votes += body.inc_votes
+    return result
+  });
+};
```

Comment:
It may not be necessary for you to check inc_votes is a number, as PSQL would throw an error. 

this model isn't actually updating your database, it is just increasing the votes on your response object.

Try to refactor this so that it updates the column in your articles table using PSQLs `UPDATE` function